.github/dependabot.yml

```diff
@@ -9,7 +9,9 @@ updates:
     schedule:
       interval: "daily"
   - package-ecosystem: "cargo"
-    directory: "/"
+    directories:
+      - "/"
+      - "/examples/wasm_github_fetch"
```

Comment:
Maybe we can remove this change since they are not related.

Reply:
I'd rather not get PRs about the WASM example dependencies _at all_. (I occasionally get security ones for npm packages, but I wish I could turn those off. It's just an example, I don't want the commit history filled with npm updates.)

Reply:
Removed Dependabot cargo for examples/wasm_github_fetch upon request (it wasn't for npm).